package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@microsoft.azure/autorest.testserver",
-  "version": "3.2.3",
+  "version": "3.3.21",
```

Comment:
@msyyc can you remove your commit for `package-lock.json`?

Reply:
reverted.